src/js/rx/containers/Active.jsx

```diff
@@ -159,7 +161,8 @@ class Active extends React.Component {
       content = (
         <p className="rx-tab-explainer rx-loading-error">
           We couldn't retrieve your prescriptions.
-          Please refresh this page or try again later.
+          Please refresh this page or try again later.&nbsp;
```

Comment:
There's an `nbsp;` here but not in the change for the History page. I'm not sure whether `ErrorMessages` should be an inline or block element, but if the space isn't necessary, it should be removed.